### DISCUSSION TITLE
Running in a turbo monorepo removes styling.

### DISCUSSION THREAD

**Discussion by jjrise** [fuma-nama/fumadocs#557] [Q&A] [answered]
I started with Fumadocs inside a turbo monorepo. I spent the better part of yesterday using fumadocs and it was awesome - no complaints. Today for some reason no matter what I do, the styling is goofed up. If I start a new project with `bun create fumadocs-app` outside of my monorepo, it runs normally and styled correctly, but as soon as I move the same app inside my monorepo `root -> apps -> docs` it has minimal styling. For more context, it was working just fine in my monorepo yesterday even with all my edits & additions. But today, even the base starter project with `bun create fumadocs-app` looks like this:

<img width="1042" alt="image" src="https://github.com/fuma-nama/fumadocs/assets/66971859/9cee5f5b-baa8-4ebc-9790-ab78f7938a97">

I feel silly asking this question because it seems so basic, but I'm spinning my wheels right now and would love some help!

Thanks!

**Comment by fuma-nama** [comment, marked as answer]
If you are using it in a monorepo, read:
https://fumadocs.vercel.app/docs/ui#fix-monorepo-styling

Make sure the files of Fumadocs UI are scanned by Tailwind CSS.

**Reply by jjrise** [reply]
ahhhh that was it!!! Thank you!
